cookie-banner-rules-list.json

```diff
@@ -7476,6 +7476,1182 @@
+        "optOut": "button#ccc-reject-settings",
+        "presence": "div#ccc"
+      },
+      "domain": "metoffice.gov.uk",
+      "cookies": {},
+      "id": "9e4d932a-eaf9-44e4-a3ac-55861d66a7c3"
+    },
+    {
+      "click": {
+        "optIn": "button#onetrust-accept-btn-handler",
+        "presence": "div#onetrust-consent-sdk"
+      },
+      "domain": "buienradar.be",
+      "cookies": {},
+      "id": "3d3ab528-e762-4d7d-a18c-865fba50d421"
+    },
+    {
+      "click": {
+        "optIn": "button#kc-acceptAndHide",
+        "presence": "div#kconsent"
+      },
+      "domain": "k-ruoka.fi",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_hjFirstSeen",
+            "value": "1"
+          },
+          {
+            "name": "_hjAbsoluteSessionInProgress",
+            "value": "1"
+          }
+        ]
+      },
+      "id": "282ff551-ce28-4b7f-9633-eaaa7ce89890"
+    },
+    {
+      "click": {
+        "optIn": "button#footer_tc_privacy_button_2",
+        "optOut": "button#footer_tc_privacy_button_3",
+        "presence": "div#tc-privacy-wrapper"
+      },
+      "domain": "cdiscount.com",
+      "cookies": {},
+      "id": "1871561d-65f8-4972-8e8a-84fa9eb704b4"
+    },
+    {
+      "click": {
+        "optIn": "button.css-fz9f1h",
+        "presence": "div#qc-cmp2-ui"
+      },
+      "domain": "dikaiologitika.gr",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "__qca",
+            "value": "P0-1708252764-1666604453009"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "addtl_consent",
+            "value": "1~"
+          },
+          {
+            "name": "Exitbee_visitsCount",
+            "value": "1"
+          }
+        ]
+      },
+      "id": "1c742d88-b3a7-43da-b205-3ba4afd92e63"
+    },
+    {
+      "click": {
+        "optIn": "button.css-k8o10q",
+        "presence": "div#qc-cmp2-ui"
+      },
+      "domain": "fantacalcio.it",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "__qca",
+            "value": "P0-1491353199-1666610397815"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "addtl_consent",
+            "value": "1~"
+          }
+        ]
+      },
+      "id": "45ce49b9-71c7-4800-9901-908f8d76807e"
+    },
+    {
+      "click": {
+        "optIn": "button.coi-banner__accept",
+        "presence": "div#coiPage-1"
+      },
+      "domain": "elkjop.no",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_hjFirstSeen",
+            "value": "1"
+          }
+        ]
+      },
+      "id": "29f00c2e-04e6-4655-9f51-6bf032abb903"
+    },
+    {
+      "click": {
+        "optIn": "button#didomi-notice-agree-button",
+        "presence": "div#buttons"
+      },
+      "domain": "naszemiasto.pl",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "_pbjs_userid_consent_data",
+            "value": "6798433657233410"
```

Comment:
Same question here. Do we want to include cookies that could be from user id?

Reply:
imo we shouldn't. This is also prone to break in the future.

Reply:
Removed the uuid cookie.